docs/building-a-form/available-form-features-and-usage-guidelines.md

```diff
@@ -419,14 +419,29 @@ For the code implementation, see the [`review` folder](../../src/js/review).
 
 ### Required checkbox before form submission
 
-Use this feature to require a user to indicate they have read terms and conditions, a privacy policy, or any other text before submitting your form. It includes a checkbox and short-form text that can include relevant links to more verbose information on separate pages on your site. To configure this feature, place a `preSubmitInfo` object in the `formConfig`:
+Use this feature to require a user to indicate they have read terms and conditions, a privacy policy, or any other text before submitting your form. It includes a checkbox and short-form text that can include relevant links to more verbose information on separate pages on your site.
+
+To configure this feature, place a `preSubmitInfo` object in the `formConfig`. These are the available options:
+<dl>
```

Comment:
This is great! In recent years, I've leaned away from `<dl></dl>` in Markdown because it's a lot more difficult to maintain (opting for plain list items with strong emphasis on the term instead). I'll leave that up to you, though.